lib/fog/aws/requests/dns/change_resource_record_sets.rb

```diff
@@ -141,6 +141,7 @@ def change_resource_record_sets(zone_id, change_batch, options = {})
           body = %Q{<?xml version="1.0" encoding="UTF-8"?><ChangeResourceRecordSetsRequest xmlns="https://route53.amazonaws.com/doc/#{@version}/">#{changes}</ChangeResourceRecordSetsRequest>}
           request({
             :body       => body,
+            :idempotent => true,
```

Comment:
As it should be performed atomically, this must be idempotent as [doc](http://docs.aws.amazon.com/Route53/latest/APIReference/api-change-resource-record-sets-example-basic.html) says.